src/util/graph-templates/template-utils.js

```diff
@@ -100,7 +100,13 @@ const util = {
       .data(() => (isMultiLevel ? head[0] : head))
       .join('th')
       .attr('rowspan', (d, i) => (i === 0 && isMultiLevel ? 2 : 1))
-      .attr('colspan', (d, i) => (isMultiLevel ? (i > 0 ? head[1].length / (head[0].length - 1) : 1) : 0))
+      .attr('colspan', (d, i) => {
```

Comment:
changes like these are because eslint does not like nested ternerys, and I agree because its harder to read them.